MIDAS/lib/TCAL9539/TCAL9539.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include<Arduino.h>
+
+struct GpioAddress {
+    int gpio_id;
+    int pin_id;
+};
+
+constexpr GpioAddress BNO85_RESET = {.gpio_id = 0, .pin_id = 0};
```

Comment:
should the bno's reset be included within the driver? This should be general enough to not have to be used specifically with midas

Reply:
oops that was left in from testing